features/specs/minderjarige-ouder.feature

```diff
@@ -0,0 +1,244 @@
+      Gegeven de 19 jaar geleden geboren vrouw 'Gerda'
+      En de 18 jaar geleden geboren man 'Aart'
+      En de 2 jaar geleden geboren persoon 'Bert' met twee ongehuwde ouders 'Gerda' en 'Aart'
+      En 2 jaar geleden is in een gerechtelijke uitspraak het gezag toegewezen aan een voogd
+      En 1 jaar geleden is in een gerechtelijke uitspraak de voogdij beëindigd
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Aart'
+
+  Regel: Als de moeder minderjarig was bij geboorte van haar kind en het kind is destijds geadopteerd en moeder is inmiddels meerderjarig, dan is er gezamenlijk gezag voor de moeder en de adoptieouder
+
+    Voorbeeld: Bij de geboorte van minderjarige was de moeder minderjarig en is de minderjarige geadopteerd en de moeder is nu meerderjarig geworden
+      Gegeven de 19 jaar geleden geboren vrouw 'Gerda'
+      En de 30 jaar geleden geboren vrouw 'Ariana'
+      En de 6 jaar geleden geboren persoon 'Bert' met één ouder 'Gerda'
+      * is 6 jaar geleden geadopteerd door Nederlandse adoptieouder 'Ariana' met een Nederlandse adoptieakte
+      Als 'gezag' wordt gevraagd van 'Bert'
+      Dan is het gezag over 'Bert' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Ariana'
+
+  Regel: Als het gezag wordt gevraagd van een minderjarige die in een gerechtelijke uitspraak meerderjarig is verklaard, dan 
+
+    Voorbeeld: Het gezag wordt gevraagd van een minderjarige met een kind
+      Gegeven de 17 jaar geleden geboren 'Zoe' met twee gehuwde ouders 'Gerda' en 'Aart'
+      En de 1 jaar geleden geboren 'Bert' met één ouder 'Zoe'
+      Als 'gezag' wordt gevraagd van 'Zoe'
+      Dan is het gezag over 'Zoe' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Ariana'
+
+    Voorbeeld: Het gezag wordt gevraagd van een minderjarige met een kind en er is een voogd aangewezen voor het kind van de minderjarige
+      Gegeven de 17 jaar geleden geboren 'Zoe' met twee gehuwde ouders 'Gerda' en 'Aart'
+      En de 1 jaar geleden geboren 'Bert' met één ouder 'Zoe'
+      En 1 jaar geleden is in een gerechtelijke uitspraak het gezag toegewezen aan een voogd
+      Als 'gezag' wordt gevraagd van 'Zoe'
+      Dan is het gezag over 'Zoe' gezamenlijk ouderlijk gezag met ouder 'Gerda' en ouder 'Ariana'
+
+    Voorbeeld: Het gezag wordt gevraagd van een minderjarige die zelf een minderjarig kind heeft en in een gerechtelijke uitspraak zij meerderjarig verklaard en heeft het gezag gekregen
+      Gegeven de 17 jaar geleden geboren 'Zoe' met twee gehuwde ouders 'Gerda' en 'Aart'
+      En de 1 jaar geleden geboren 'Bert' met één ouder 'Zoe'
+      En 1 jaar geleden is in een gerechtelijke uitspraak het gezag toegewezen aan 'Zoe'
```

Comment:
Is zoe nou meerderjarig verklaard of is aan Zoe het gezag toegewezen. Welke uitspraak is het?
Ik denk dat het goed is om dat onderscheid te maken.